health/egov-persister/plan-configuration-persister.yml

```diff
@@ -0,0 +1,78 @@
+serviceMaps:
+  serviceName: plan-service
+  mappings:
+    - version: 1.0
+      name: plan-configuration
+      description: Persists plan configuration details
+      fromTopic: plan-config-create-topic
+      isTransaction: true
+      isAuditEnabled: true
+      module: PLAN
+      objecIdJsonPath: $.PlanConfiguration.id
```

Comment:
change this to the path after the base path to access the attribute for e.g. here it should be $.id

Reply:
same should be done for tenantIdJsonPath, userUuidJsonPath and transactionCodeJsonPath